python/lsst/pipe/tasks/assembleCoadd.py

```diff
@@ -605,3 +622,298 @@ def makeDataRefList(self, namespace):
             )
             self.refList.append(dataRef)
 
+def countMaskFromFootprint(mask, footprint, bitmask, ignoreMask):
+    """Function to count the number of pixels with a specific mask in a footprint
+    """
+    bbox = footprint.getBBox()
+    bbox.clip(mask.getBBox(afwImage.PARENT))
+    fp = afwImage.MaskU(bbox)
+    subMask = mask.Factory(mask, bbox, afwImage.PARENT)
+    afwDet.setMaskFromFootprint(fp, footprint, bitmask)
+    return numpy.logical_and((subMask.getArray() & fp.getArray()) > 0,
+                             (subMask.getArray() & ignoreMask) == 0).sum()
+
+
+class SafeClipAssembleCoaddConfig(AssembleCoaddConfig):
+    clipDetection = pexConfig.ConfigurableField(target=SourceDetectionTask,
+                                      doc="Detect sources on difference between unclipped and clipped coadd")
+    minClipFootOverlap = pexConfig.Field(
+        doc = "Minimum fractional overlap of clipped footprint with visit DETECTED to be clipped",
+        dtype = float,
+        default = 0.65
+    )
+    minClipFootOverlapSingle = pexConfig.Field(
+        doc = "Minimum fractional overlap of clipped footprint with visit DETECTED to be " \
+            "clipped when only one visit overlaps",
```

Comment:
Add a couple of extra spaces to help the strings line up, here and in the other `Field`s defined below.